### PR TITLE
3708 workbench guidance

### PR DESCRIPTION
**Pull request by soyarsauce**
(partially resolves):
https://github.com/terriajs/terriajs/issues/3708

Addresses:
https://github.com/terriajs/de-australia-map/issues/14

### CODE REVIEW

File: lib/ReactViews/Guide/satellite-guidance.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    body:
+      "A super-quick guide to getting the most out of historical satellite imagery.",
+    prevText: "No thanks",
+    nextText: "View guide"
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-zoom.jpg"),
+    title: "Where is my satellite imagery?",
+    body:
+      "You may need to adjust your zoom level to locate near real-time satellite imagery on the map.",
```
Comment (rowanwins) [inline lib/ReactViews/Guide/satellite-guidance.js]: Suggest rewording from 
"You may need to adjust your zoom level to locate near real-time satellite imagery on the map.",
to 
"You may need to adjust your zoom level to make satellite imagery visible on the map.",

Rationale being
- it's not just "near real-time" imagery effected
- and zooming in doesn't "locate" the imagery

File: lib/ReactViews/Guide/satellite-guidance.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    imageSrc: require("../../../wwwroot/images/guides/satellite-location.jpg"),
+    title: "Filtering imagery by location",
+    body:
+      "Satellite imagery may not always be available at your preferred time and location. If you only want to view imagery for a particular location, click ‘filter by location’ in your data workbench."
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-styles.jpg"),
+    title: "How do I view false-colour imagery?",
+    body:
+      "You can apply a range of false-colour styles to satellite imagery by clicking on the styles dropdown in your data workbench.",
```
Comment (rowanwins) [inline lib/ReactViews/Guide/satellite-guidance.js]: Suggest changing
"You can apply a range of false-colour styles to satellite imagery by clicking on the styles dropdown in your data workbench.",
to
"You can apply a range of styles, such as false-colour, to satellite imagery by clicking on the styles dropdown in your data workbench.",

File: lib/ReactViews/Guide/satellite-guidance.js (hunk 1 above)
Comment (soyarsauce) [reply lib/ReactViews/Guide/satellite-guidance.js]: Thanks that's a better, more generic copy that applies!

File: lib/ReactViews/Guide/satellite-guidance.js (hunk 2 above)
Comment (soyarsauce) [reply lib/ReactViews/Guide/satellite-guidance.js]: 👍

**Review by rowanwins — APPROVED** [review]
Looks good to me @soyarsauce
